srcalc/src/main/java/gov/va/med/srcalc/web/controller/ReferenceDataController.java

```diff
@@ -50,6 +50,14 @@ public ReferenceDataController(final ReferenceDataService procedureDao)
             jsonProcedure.put("shortDescription", p.getShortDescription());
             // Use Float.toString() to send RVU as a string.
             jsonProcedure.put("rvu", Float.toString(p.getRvu()));
+            if(p.isEligible())
```

Comment:
This translation from "eligible" to "excluded" is awkward. Can we just standardize on using "eligible" throughout the app?